app/models/choice.rb

```diff
@@ -0,0 +1,7 @@
+class Choice < ApplicationRecord
+    belongs_to :word, foreign_key: "word_id"
```

Comment:
remove `foreign_key: "word_id"` since it looks like Ruby on Rails and manual putting of foreign key here will cause conflict